app/main.py

```
@@ -8,6 +8,9 @@
 
 from app.config import settings
 from app.logging_config import setup_logging
+from contextlib import asynccontextmanager
+from app.services import connect_db, disconnect_db
+from app.config import settings, TORTOISE_ORM
```

Ruff (0.11.9)

Redefinition of unused `settings` from line 9

Remove definition: `settings`

(F811)